index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
   <body>
-    <nav>
-      <a class="logo" href="#">
+    <header class="header">
+      <a class="header--logo" href="#">
         <img src="images/keys.png" alt="keyboard" />
       </a>
-      <ul class="nav-links-cont">
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+      <nav class="header--nav">
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-call" href="#">
-          <button>Button</button>
+
+        <a class="button--small header--button">
+          Button
         </a>
-      </ul>
-    </nav>
-    <div class="top-cont">
+      </nav>
+    </header>
+
+    <div class="main">
       <div class="gradient-overlay"></div>
-      <div class="image-and-call-cont">
-        <div class="call-cont">
-          <div class="call-cont-inner">
-            <h1>Web development at its finest</h1>
-            <a href="#">
-              <button>Button</button>
-            </a>
+
+      <div class="hero">
```

Comment:
I tried to keep the names short, image-and-call-cont is just a lot to have to type. We gotta save our poor little developer fingers! 💅 

Plus with a class like .hero, you immediately know what that element is when you see it, it's a hero.

In comparison, an image-and-call-cont could be any number of things!